src/js/components/Select/doc.js

```diff
@@ -23,6 +23,27 @@ export const doc = Select => {
       { active, disabled, selected } keys indicating the current state
       of the option.`,
     ),
+    clear: PropTypes.oneOfType([
+      PropTypes.bool,
+      PropTypes.shape({
+        position: PropTypes.oneOf(['top', 'bottom'])
+          .description(
+            `Add a clear value to the top or at the bottom of the
+            container. By default no clear option is present.`,
+          )
+          .defaultValue('top'),
+        label: PropTypes.string
+          .description('Label for the clear selection item')
+          .defaultValue('Clear selection'),
+        render: PropTypes.func.description(
+          `Render function to customize clear value item.
+          It receives as a parameter the callback to call
+          when clicking on the item`,
+        ),
+      }),
+    ])
+      .description(`Whether to provide button to clear selection.`)
```

Comment:
```suggestion
      .description(`Whether to provide a button option to clear selections.`)
```
Does that make more sense?